iree/compiler/Conversion/HLOToLinalg/HLOToLinalgOnTensors.cpp

```diff
@@ -346,15 +348,25 @@ struct ConvertHLOToLinalgOnTensorsPass
 struct ConvertHLOToLinalgOnTensorsPassExperimental
     : public ConvertHLOToLinalgOnTensorsPass {
   ConvertHLOToLinalgOnTensorsPassExperimental()
-      : ConvertHLOToLinalgOnTensorsPass(true){};
+      : ConvertHLOToLinalgOnTensorsPass(true) {}
```

Comment:
Ugh I should've ignored the lint warning :-D I'm kind of worried that this will be somehow dependended on by something internally, so would prefer not to mix it up with this change.